R/postProcess.R

```diff
@@ -1330,12 +1330,52 @@ postProcessAllSpatial <- function(x, studyArea, rasterToMatch, useCache, filenam
       extRTM <- NULL
       crsRTM <- NULL
     }
+    useBuffer <- FALSE
+    bufferSA <- FALSE
+
+    if (class(x) == "RasterLayer") {
+      #if all CRS are projected, then check if buffer is necessary
+      projections <- sapply(list(x, studyArea, crsRTM), FUN = sf::st_is_longlat)
+      projections <- na.omit(projections)
+      if (!any(projections)) {
+        if (is.null(rasterToMatch)) {
```

Comment:
both conditions set `useBuffer = TRUE`, so why are we testing in the first place?